Softeq.XToolkit.WhiteLabel.Forms/Converters/StringToUpperCaseConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace Softeq.XToolkit.WhiteLabel.Forms.Converters
+{
+    /// <summary>
+    ///     Converts a string to UpperCase string.
+    /// </summary>
+    public class StringToUpperCaseConverter : IValueConverter
```

Comment:
Also can be added `ToUpper` converter separately

Reply:
Renamed.
@wcoder Please, explain what do you mean by 

> Also can be added ToUpper converter separately

Reply:
It may also be useful to have an opposite converter (separately)